### PR TITLE
AkihiroSuda/containerd-fuse-overlayfs -> containerd/fuse-overlayfs-snapshotter

### PR DESCRIPTION
**Pull request by AkihiroSuda**
#### Proposed Changes ####

Move `github.com/AkihiroSuda/containerd-fuse-overlayfs` to https://github.com/containerd/fuse-overlayfs-snapshotter

#### Types of Changes ####

Refactoring

#### Verification ####

`k3s server --rootless --snapshotter=fuse-overlayfs`

### CODE REVIEW

**Review by brandond — APPROVED** [review]

File: go.mod
Comment (brandond) [inline go.mod:70]: I don't think this is going to work since we have a replace line up above: `github.com/containerd/containerd => github.com/k3s-io/containerd v1.4.4-k3s1`

**Review by brandond — CHANGES_REQUESTED** [review]

**Review by briandowns — APPROVED** [review]

File: go.mod
Comment (dweomer) [reply go.mod:70]: we are not moving to containerd 1.5.x until it is out of beta afaik

**Comment by dweomer** [comment]
This looks to be adopting containerd 1.5.x which we will hold off on until it is out of beta.

File: go.mod
Comment (AkihiroSuda) [reply go.mod:70]: This doesn't upgrade containerd to 1.5, but `go mod tidy` fills a dummy `v1.5.0-beta.4` value here.
The actual version is `replace`d to v1.4.4-k3s1.

**Comment by AkihiroSuda** [comment]
No, this doesn't upgrade containerd https://github.com/k3s-io/k3s/pull/3106#discussion_r600142810

**Review by brandond — APPROVED** [review]

**Comment by brandond** [comment]
I'll merge this when the conflict is resolved. I took a shot at it myself but it looks like 'Allow edits from maintainers' isn't enabled for this PR.

**Comment by AkihiroSuda** [comment]
Rebased

**Pull request merged**
